exporter/elasticsearchexporter/README.md

```diff
@@ -246,6 +246,23 @@ The metric types supported are:
 - Exponential histogram (Delta temporality only)
 - Summary
 
+## Exporting profiles
+
+Profiles support is currently in development, and should not be used in
+production. Profiles only support the OTel mapping mode.
+
+Example:
+
+```yaml
+[...]
+service:
+	extensions: [basicauth]
+	pipelines:
+		profiles:
+			receivers: [otlp]
+			exporters: [elasticsearch]
```

Comment:
I was thinking of documenting the working configuration of the exporter for profiles, including setting the right mapping mode.
```suggestion
exporters:
  elasticsearch:
    endpoint: https://elastic.example.com:9200
    mapping:
      mode: otel
```